src/caret_analyze/runtime/path.py

```diff
@@ -24,7 +24,7 @@
 from ..common import Summarizable, Summary, Util
 from ..exceptions import Error, InvalidArgumentError, InvalidRecordsError
 from ..record import Columns, RecordsFactory
-from ..record.record import merge, merge_sequential, RecordsInterface
+from ..record.record_utils import merge, merge_sequential, RecordsInterface
```

Comment:
```suggestion
from ..record.record_utils import merge, merge_sequential
from ..record.interface import RecordsInterface
```

Reply:
@h-suzuki-isp 
Sorry. This fix caused the import order to be incorrect. Please correct it.

Reply:
Ok, fixed in https://github.com/tier4/caret_analyze/pull/493/commits/b44f40cef3d15a08b5194bbf1fe581eabc49419f.